papierowyRPG_API/Models/Items.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace papierowyRPG_API.Models
+{
+    public class Items
```

Comment:
Let's call them singular, like `Item`, as this will represent one item. Do it also in other places